models/news.model.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection.js");
+
+const fetchTopics = () => {
+  return db.query("SELECT * FROM topics").then(({ rows: topics }) => {
+    return { topics };
+  });
+};
+
+const fetchArticle = (id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [id])
+    .then(({ rows: article }) => {
+      if (article.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
```

Comment:
This will do on it's own - you don't also need the checkArticleExists function! If the article id does not exist - it will return an empty array and, the error will be thrown here. You will need to check if something exists later in the sprint, but not yet!